pkgs/desktops/gnome/apps/gnome-clocks/default.nix

```diff
@@ -1,34 +1,36 @@
-{ stdenv
-, lib
-, fetchurl
-, meson
-, ninja
-, gettext
-, pkg-config
-, wrapGAppsHook4
-, itstool
-, desktop-file-utils
-, vala
-, libxml2
-, gtk4
-, glib
-, sound-theme-freedesktop
-, gsettings-desktop-schemas
-, gnome-desktop
-, geocode-glib_2
-, gnome
-, gdk-pixbuf
-, geoclue2
-, libgweather
-, libadwaita
+{
+  stdenv,
+  lib,
+  fetchurl,
+  meson,
+  ninja,
+  gettext,
+  pkg-config,
+  wrapGAppsHook4,
+  itstool,
+  desktop-file-utils,
+  vala,
+  libxml2,
+  gtk4,
+  glib,
+  sound-theme-freedesktop,
```

Comment:
This is not necessary any more IIRC.

Reply:
I don't see any sound themes in `/nix/store/mcvwh9a7mgk2ld7ica4kdbf8qs54r527-gnome-clocks-46.0/bin/gnome-clocks`.

But I suppose it should be fine to remove this since I can see these in 

![image](https://github.com/NixOS/nixpkgs/assets/50095635/94d6d929-eba7-4b51-be56-f128e8c87110)
Does NixOS by default links sound themes in `/run` or is it controlled by some option?

Reply:
Looks like there is [`xdg.sounds.enable`](https://search.nixos.org/options?channel=unstable&show=xdg.sounds.enable&from=0&size=50&sort=relevance&type=packages&query=sound), enabled by default. So let's remove this?

Reply:
Signs themes are mostly dead nowadays. IIRC, the app switched to playing a built in ogg file.